### PR TITLE
Rename the MicroProfile Metric Bundles to use the `io.openliberty.*` and `io.openliberty.*.internal.*` convention.

### PR DESCRIPTION
**Pull request by Channyboy**
Rename the MicroProfile Metric Bundles to use the `io.openliberty.*` and `io.openliberty.*.internal.*` convention.
#build

### CODE REVIEW

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
Requested a new build with some custom changes here:
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_grQlMNjREeqBAvt2Ajbd0w&tab=com.ibm.team.build.web.ui.internal.editors.result.overview.OverviewPage

The automatic build is still requesting FAT buckets to run with the old naming pattern which causes failures:
`com.ibm.ws.microprofile.metrics.3.x.monitor_fat` and 
`com.ibm.ws.microprofile.metrics.3.0_fat_tck`

This custom build now has been renamed to run only `io.openliberty` pattern related metrics 3.0 fats.

**Comment by Channyboy** [comment]
FAT failures are unrelated.
They are due to connection time outs and excessive trace volume on fat buckets of unrelated components (i.e components that do not depend on MicroProfile Metircs or MicoProfile in any way)

There were FAT issues regarding the build trying to execute the old `com.ibm.` MP Metrics 3.0 fats in that it could not find them anymore (since they have been deleted).

The build also ran the renamed `io.openliberty` MP Metrics 3.0 fats without issue.

File: dev/io.openliberty.microprofile.metrics.internal.cdi.3.0/.project [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>com.ibm.ws.microprofile.metrics.3.0.monitor</name>
+	<name>io.openliberty.microprofile.metrics.internal.cdi.3.0</name>
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.metrics.internal.cdi.3.0/.project:3]: This doesn't look right.  Rename from `com.ibm.ws.microprofile.metrics.3.0.monitor` to `io.openliberty.microprofile.metrics.internal.cdi.3.0`?

File: dev/io.openliberty.microprofile.metrics.internal.public/.project [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>com.ibm.ws.microprofile.metrics.3.x.monitor_fat</name>
+	<name>io.openliberty.microprofile.metrics.internal.public</name>
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.metrics.internal.public/.project:3]: Not sure this is a git problem or the file is wrongly renamed.

File: dev/com.ibm.websphere.appserver.features/visibility/public/mpMetrics-3.0/io.openliberty.mpMetrics-3.0.feature [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 IBM-ShortName: mpMetrics-3.0
 Subsystem-Name: MicroProfile Metrics 3.0
--features=com.ibm.websphere.appserver.org.eclipse.microprofile.metrics-3.0, \
+-features=io.openliberty.org.eclipse.microprofile.metrics-3.0, \
  com.ibm.websphere.appserver.cdi-2.0; ibm.tolerates:=1.2,\
  com.ibm.websphere.appserver.javax.annotation-1.3; ibm.tolerates:=1.2, \
  com.ibm.websphere.appserver.restHandler-1.0, \
  com.ibm.websphere.appserver.monitor-1.0, \
  com.ibm.websphere.appserver.servlet-4.0; ibm.tolerates:=3.1,\
  com.ibm.websphere.appserver.mpConfig-1.4; ibm.tolerates:=1.3
```
Comment (fmhwong) [inline dev/com.ibm.websphere.appserver.features/visibility/public/mpMetrics-3.0/io.openliberty.mpMetrics-3.0.feature:15]: Probably should use mpConfig 2.0-m1

File: dev/io.openliberty.microprofile.metrics.internal.3.0/bnd.bnd [hunk 4]
```diff
@@ -60,7 +60,7 @@ WS-TraceGroup: METRICS
 	com.ibm.websphere.org.osgi.service.component;version=latest,\
 	com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
 	com.ibm.websphere.org.eclipse.microprofile.config.1.4;version=latest,\
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.metrics.internal.3.0/bnd.bnd:62]: Probably should use mpConfig 2.0-m1

File: dev/io.openliberty.microprofile.metrics.internal.cdi.3.0/bnd.bnd [hunk 5]
```diff
@@ -55,8 +55,8 @@ WS-TraceGroup: METRICS
 	com.ibm.websphere.org.eclipse.microprofile.config.1.4;version=latest,\
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.metrics.internal.cdi.3.0/bnd.bnd:55]: Probably should use mpConfig 2.0-m1

File: dev/io.openliberty.microprofile.metrics.internal.public/.project (hunk 2 above)
Comment (Channyboy) [reply dev/io.openliberty.microprofile.metrics.internal.public/.project:3]: Right, should be a git issue.
Projects were copied/duplicated then renamed.
Then the original `com.ibm.*` projects were deleted at last.

**Comment by Channyboy** [comment]
#build
#run-libby-bot

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
Squashed.

**Comment by Channyboy** [comment]
The MP Metrics 3.0 bundles are are isolated in that no other bundles depend on it.
The `io.openliberty` MP Metrics 3.0 FAT TCK all pass successfully.

**Comment by Channyboy** [comment]
#run-libby-bot (after squash)

**Comment by Channyboy** [comment]
The MP Metrics 3.0 bundles are are isolated in that no other bundles depend on it.
The `io.openliberty` MP Metrics 3.0 FAT TCK all pass successfully in the builds.

**Pull request merged**